eve/main.yml

```diff
@@ -90,6 +90,11 @@ models:
   - ShellCommand: &s3-log
```

Comment:
Would replace this correct?

Reply:
Exactly. I think I'll replace this step by an upload. As we won't need it anymore